src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -3216,7 +3216,25 @@ class C
                 Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "=> 3").WithArguments("System.Runtime.CompilerServices.IAsyncStateMachine", "SetStateMachine").WithLocation(4, 25));
         }
 
-        [Fact]
+        public string AsyncBuilderCode(string builderTypeName, string tasklikeTypeName, string genericTypeParameter = null, bool isStruct = false)
```

Comment:
Fixed in https://github.com/dotnet/roslyn/pull/13405/commits/f2cdba83c83d34ddac4875def41674d352fc5b7e#diff-c55f1eb2556bfc07b602d0647778d699R3219